src/main/java/com/dumbdogdiner/stickyapi/common/util/TimeUtil.java

```diff
@@ -17,11 +18,13 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
+ * TODO Cleanup to use the Java instants stuff
  * Utility methods for dealing with time and duration parsing.
  */
 public class TimeUtil {
     private static final SimpleDateFormat sdf = new SimpleDateFormat("EEE, MMM d yyyy HH:mm:ss");
 
+    //TODO convert this stuff to use an enum so it's nicer
```

Comment:
This would make it somewhat annoying to access at runtime. Recommend keeping this as a hash map.